Backend/app/rider-platform/rider-app/Main/src/API.hs

```diff
@@ -64,6 +65,7 @@ mainServer :: FlowServer MainAPI
 mainServer =
   UI.handler
     :<|> Beckn.handler
+    :<|> const Beckn.handler
```

Comment:
why its `const`? you never use `Capture "merchantId"`, so why its added?

Reply:
In BAP side it's not needed as of now but in BPP side we want different bapId so that we can show in BPP side information related to which BAP the search request has come from.